### ISSUE TITLE
Msg Me Here

### ISSUE THREAD

**Issue by Glatrix** [Glatrix/MESSAGE-ME-HERE#1]

**Comment by steam-lil-meatball** [comment]
Yo, I just found you on steam I just wanted to say I couldnt add you, so if you can add me lil meatball that would be good
you were [Mod] Bo right?

**Comment by karmakittenx** [comment]
I would appreciate if you removed your WraithGUI repos. All you done is remove the credit and upload as your own. You don't even have the modified DLL to make it function so I would greatly appreciate if you removed it until it's truly ready. Thanks!

**Comment by karmakittenx** [comment]
Although, I am possibly interested on working with PhasMods if you possibly consdier?

**Comment by Glatrix** [comment]
Yes you can join. I had no intent of stealing. I can remove the repos if you would like.
I could not figure out how to compile it. 
My apologies.

**Comment by karmakittenx** [comment]
Don't worry about it man, i just talked with Cr4nkSt4r and he said you were only using it as a base to learn about MelonMods. I'm always glad to help I just thought you were stealing my work is all :)

**Comment by Rencify** [comment]
Hey, do you have the updated beta version cheat table for Phasmophobia? Would be great if you could send me it if possible, thank you!

**Comment by Glatrix** [comment]
nope nope and nope.
1. I stated on Unkowncheats that I will no longer support beta. Every time I release beta hacks, they get patched. I will
release them to the public once the update goes live. no exceptions.

2. My Parents don't allow social media.

Sorry!
